packages/react-examples/src/react/TimePicker/TimePicker.Example.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
```

Comment:
@CheerfulSatchel are you planning to follow up with a PR to get the TimePicker examples up on https://developer.microsoft.com/en-us/fluentui#/controls/web? This would make this control broadly discoverable so that anyone using Fluent UI can benefit from it.